core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.html

```diff
@@ -0,0 +1,15 @@
+<form class="p-fluid" *ngIf="form" [formGroup]="form">
+    <ng-container *ngFor="let form of formData">
```

Comment:
should this be: `let row of formData`? And then below `let column of row.columns`  that way we keep the row -> column hierarchy